bundle/paths/paths.go

```diff
@@ -0,0 +1,39 @@
+package paths
+
+import (
+	"strings"
+
+	"github.com/databricks/cli/bundle/config"
+	"github.com/databricks/cli/bundle/libraries"
+)
+
+func CollectUniquePaths(workspace config.Workspace) []string {
```

Comment:
This can be a bit more specific, or as a comment.

```suggestion
func CollectUniqueWorkspacePathPrefixes(workspace config.Workspace) []string {
```